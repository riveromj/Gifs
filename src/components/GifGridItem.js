import React from 'react'
import PropTypes from 'prop-types';


export const GifGridItem = ({title, url}) => {

    return (
        <div className="card animate__backInDown">
           <img className='gif' src={url} alt={title}/>
           <p>{title}</p>
        </div>
    )
}

GifGridItem.protoTypes={
    title:PropTypes.string.isRequired,
    url:PropTypes.string.isRequired
}