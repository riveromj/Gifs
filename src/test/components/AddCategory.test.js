import { shallow } from 'enzyme'
import React from 'react'
import { AddCategory } from '../../components/AddCategory'

describe('pruebas de AddCategory', () => {
    const setCategories = ()=>{};
    const wrapper = shallow(<AddCategory setCategories={setCategories}/>)

    test('que el componete se muestre de forma correcta ', () => {
        
       
        
        expect(wrapper).toMatchSnapshot();
    })

    ///simular interacciones de usuarios.
    test(' ', () => {
        
    })
    
    
    
})

