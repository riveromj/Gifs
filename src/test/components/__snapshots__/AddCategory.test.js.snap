// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas de AddCategory que el componete se muestre de forma correcta  1`] = `
<form
  onSubmit={[Function]}
>
  <h2>
    search
  </h2>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
